<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TxtToSpeech</class>
 <widget class="QDialog" name="TxtToSpeech">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>语音合成系统</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>261</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>语音合成系统V0.0.1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>是否保存MP3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hec">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>语音合成</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_voice_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>讯飞小燕</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞许久</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小萍</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小婧</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞许小宝</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小东</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小王</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞萌萌</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小梅</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小瑞</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小梦</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小施</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小强</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞一峰</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小媛</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞晓倩</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小莹</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小乔</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小瑶</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞萌萌-悲伤</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小春</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞马叔</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小蓉</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞芳芳</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞晓峰</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小魏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞楠楠</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小肥</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小雪</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞晓琳</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小包</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞小华</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞宋宝宝</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞玉儿</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讯飞萌萌-高兴</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_load_file">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加载TXT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>发音人：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_more">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>太少,要更多</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tips">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_tips">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>134</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>306</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;软件&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;维护时间&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;9&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;月&lt;/span&gt;，&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;过期不维护&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
